ta/sdp_basic/ta_sdp_basic.c

```diff
@@ -256,7 +256,7 @@ static TEE_Result cmd_invoke(uint32_t nParamTypes,
 	        res = TEE_OpenTASession(&uuid, TEE_TIMEOUT_INFINITE, 0, NULL,
 					&sess, &ret_orig);
 		if (res != TEE_SUCCESS) {
-	                EMSG("SDP basic test TA: TEE_OpenTASession() FAILED \n");
+	                EMSG("SDP basic test TA: TEE_OpenTASession() FAILED ");
```

Comment:
```suggestion
	                EMSG("SDP basic test TA: TEE_OpenTASession() FAILED");
```
Also remove trailing whitespace.